lib/addons/aws-for-fluent-bit/index.ts

```diff
@@ -45,10 +45,7 @@ export class AwsForFluentBitAddOn extends HelmAddOn {
 
     deploy(clusterInfo: ClusterInfo): Promise<Construct> {
         const cluster = clusterInfo.cluster;
-
-        // Create the FluentBit namespace.
         const namespace = this.options.namespace;
-        createNamespace(this.options.namespace!, cluster, true);
```

Comment:
it is going to fail if namespace is custom, not kube-system. let's add a flag to like
```
createNamespace: true | false
```
if true, let's create namespace and add dependencies.

Reply:
addressed.